packages/react-network/src/server.ts

```diff
@@ -29,7 +29,10 @@ export function applyToContext<T extends Context>(
     // only set cookies that have been changed
     // decode URI as the manager's cookie value is also decoded
     const rawCookieValue = ctx.cookies.get(cookie);
-    if (rawCookieValue && decodeURIComponent(rawCookieValue) === value) {
+    if (
+      typeof rawCookieValue !== 'undefined' &&
```

Comment:
is there a reason we have to use typeof instead of what we usually do that checks for both null and undefined (`!= null`)?

Reply:
😂  I ask similar question cuz this syntax was confusing me. 
as it turn out thou, rawCookieValue is type as string | undefined.

Reply:
`!= null` works too I have no string preference. I'll change it